### PR TITLE
[#27] More descriptive releases

### LINKED ISSUE #27: Document release artifacts - purpose, source, naming/versioning
There are a lot of components in tezos. Combined with various packaging flavors this produces a lot of release artifacts - individual binaries, archives containing collections of binaries etc. It would be helpful to clearly document each release artifact: what does it contain or represent, why is it named the way it's named (explain inclusion or omission of git revision, Tezos protocol version/hash, correspondence to upstream branch names and other relevant details).

This can be achieved either by listing each and every release artifact in README along with a description or by describing general approach and naming patterns in relation to upstream source and branching/versioning/release structure (such as it is).

### PR DESCRIPTION
**Pull request by rvem**
## Description
Problem: One can be confused by the huge amount of release assets.

Solution: Add some assets description in the README and refer to
this section in release notes.

## Related issue(s)

Resolves #27

#### Related changes (conditional)

- [x] I checked whether I should update the [README](../tree/master/README.md)

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -137,6 +136,23 @@ let
...
 
+  releaseFile = writeTextFile {
+    name = "release-notes.md";
+    text = ''
+      Automatic release on ${builtins.substring 0 8 timestamp}
+
+      This release contains assets based on [${babylonnet.rev} revision](https://gitlab.com/tezos/tezos/tree/${babylonnet.rev}) of babylonnet branch and
+      [${mainnet.rev} revision](https://gitlab.com/tezos/tezos/tree/${mainnet.rev}) of mainnet branch from [tezos repository](https://gitlab.com/tezos/tezos/).
+
+      For more information about release assets see [README section](https://github.com/serokell/tezos-packaging#obtain-binaries-or-packages-from-github-release).
```
Comment (gromakovsky) [inline default.nix]: I think for actual releases (not auto pre-releases) the link should be https://github.com/serokell/tezos-packaging/blob/202001221235/README.md#obtain-binaries-or-packages-from-github-release. So I propose to change it to https://github.com/serokell/tezos-packaging/blob/autorelease/README.md#obtain-binaries-or-packages-from-github-release here and add a comment saying that one should change `autorelease` to a new tag when one makes a release.

File: README.md [hunk 2]
```diff
@@ -18,8 +18,37 @@ Recomended way to get these binaries is to download them from assets from github
 Go to the [latest release](https://github.com/serokell/tezos-packaging/releases/latest)
 and download desired assets.
 
+We provide both individual and archived binaries for mainnet and babylonnet versions
+in order to be able to have static links for these binaries, also make it convenient
+to download single binary or obtain all binaries in one click.
+
+Individual binaries (as well as Ubuntu/Fedora packages) have `mainnet/babylonnet` prefix
```
Comment (gromakovsky) [inline README.md]:
```suggestion
Individual binaries (as well as Ubuntu/Fedora packages) have `mainnet/babylonnet` suffix
```

Comment (rvem) [reply README.md]: mda

File: default.nix [hunk 3]
```diff
@@ -137,6 +136,25 @@ let
...
 
+  releaseFile = writeTextFile {
+    name = "release-notes.md";
+    text = ''
+      Automatic release on ${builtins.substring 0 8 timestamp}
+
+      This release contains assets based on [${babylonnet.rev} revision](https://gitlab.com/tezos/tezos/tree/${babylonnet.rev}) of babylonnet branch and
+      [${mainnet.rev} revision](https://gitlab.com/tezos/tezos/tree/${mainnet.rev}) of mainnet branch from [tezos repository](https://gitlab.com/tezos/tezos/).
+      <!--
+      When making a new release, replace `autorelease` with actual release tag:
```
Comment (gromakovsky) [inline default.nix]: Oh, sorry, the tag turned out to be `auto-release` (with a dash). Please change here and below (in the link).

**Pull request merged**